app/upgrades/v13/upgrades.go

```diff
@@ -98,7 +98,6 @@ func migrateNextPoolId(ctx sdk.Context, gammKeeper *gammkeeper.Keeper, swaproute
 	// N.B: pool id in gamm is to be deprecated in the future
 	// Instead,it is moved to swaprouter.
 	nextPoolId := gammKeeper.GetNextPoolId(ctx)
-	gammKeeper.SetPoolCount(ctx, nextPoolId-1)
 	swaprouterKeeper.SetNextPoolId(ctx, nextPoolId)
```

Comment:
Without going through the very fun e2e debugging process yet, my first reaction to the e2e error is the nextPoolID is not getting properly set

Reply:
@czarcas7ic hoping this: [0cf44a3](https://github.com/osmosis-labs/osmosis/pull/3492/commits/0cf44a3cef30d26bdc72da8bc1e3d5467c29ce20) fixed e2e. It was calling the removed query